Process.h

```diff
@@ -335,6 +335,8 @@ void Process_makeCommandStr(Process* this, const struct Settings_ *settings);
 
 void Process_writeCommand(const Process* this, int attr, int baseAttr, RichString* str);
 
+char processStateChar(ProcessState state);
+
```

Comment:
Since this is being made available globally now, I think a name matching the rest of the header declarations would be ideal (e.g. Process_stateChar).